types/lib/utils/import-href.d.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   lib/utils/import-href.html
+ */
+
+/// <reference path="boot.d.ts" />
+
+declare namespace Polymer {
+
+
+  /**
+   * Convenience method for importing an HTML document imperatively.
+   *
+   * This method creates a new `<link rel="import">` element with
+   * the provided URL and appends it to the document to start loading.
+   * In the `onload` callback, the `import` property of the `link`
+   * element will contain the imported document contents.
+   *
+   * @returns The link element for the URL to be loaded.
+   */
+  function importHref(href: string, onload?: Function, onerror?: Function, optAsync?: boolean): HTMLLinkElement;
```

Comment:
It'd be nice to have better types for the callbacks. Issue in Polymer core?

Reply:
Yep, would just need to update the `@params` in `import-href.html`. Punting for now.

Reply:
Funny, I filed an issue a ways back and it got comments recently: https://github.com/Polymer/polymer/issues/4603